topo_processor/data/data_transformers/data_transformer_imagery_historic.py

```diff
@@ -23,18 +23,26 @@ def is_applicable(self, item: Item) -> bool:
     async def transform_data(self, item: Item) -> None:
         cog_asset_list = []
         for asset in item.assets:
-            if is_tiff(asset.source_path):
-                start_time = time_in_ms()
-                output_path = os.path.join(item.collection.get_temp_dir(), f"{ulid.ULID()}.tiff")
+            if not is_tiff(asset.source_path):
+                return
+
+            start_time = time_in_ms()
+            output_path = os.path.join(item.collection.get_temp_dir(), f"{ulid.ULID()}.tiff")
+
+            if asset.source_path.startswith("s3://"):
```

Comment:
I think this change should be handled inside the `create_cog`

and we should have a `path.is_s3(source_path)` function somewhere rather than doing startswith `s3://`